raspberry_pi/voc_co2/CCS811_logger.py

```diff
@@ -0,0 +1,36 @@
+from time import sleep, time
+from Adafruit_CCS811 import Adafruit_CCS811
+import csv
+import os.path 
+
+def write2csv(data):
+	dataCSV = open('CCS811_log.csv', 'a')
```

Comment:
Let's call it `pi_ccs811_readings.csv`, to avoid 1) someone thinking this is for the arduino since it defaults to that on most websites, 2) not be confused by error logs, which is when I can imagine someone looking at them locally instead of directly into the database.